de.dlr.sc.virsat.model.extension.requirements/concept/concept.concept

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+Concept de.dlr.sc.virsat.model.extension.requirements displayname "Requirements"  description "Extension for Requirement Specification" version 0.1 beta 
+{
+
+	Category RequirementsSpecification {
+		Applicable For All;
+		StringProperty fileName;
+		Type requirements [] of Category Requirement;
+		Type typeDefinitions [] of Category RequirementType;
+	}	
+	
+	Category RequirementsConfiguration {
+		Applicable For All;
+		StringProperty fileName;
+		Type typeDefinitions [] of Category RequirementType;
+	}
+	
+	Category RequirementObject description "Interface for specification objects"{
+		IsAbstract;
+	}
+	
+	Category RequirementGroup extends RequirementObject{
+		
+		Type children [] of Category RequirementObject;
+		
+	}
+	
+	Category Requirement extends RequirementObject{
+		
+		Reference reqType of Type RequirementType;
+		Type elements [] of Category AttributeValue;
+		EnumProperty status values [Open = 0, FullyCompliant = 1, PartlyCompliant = 2];
+		
+	}
+	
+	Category AttributeValue{
+		
+		Reference attType of Type RequirementAttribute;
+		StringProperty value;
+		
+	}
+	
+	Category RequirementType description "Definition of a requirement type"{
+		
+		
+		Type attributes[] of Category RequirementAttribute;
+		
+	}
+	
+	Category RequirementAttribute description "Attribute definition for requirements"{
+		
+		EnumProperty  type values [Boolean = 0, Date = 1, Enumeration = 2, Integer= 3, Real= 4, String = 5] default String;
+		Type enumeration of Category EnumerationDefinition;
+		
+	}
+	
+	Category EnumerationDefinition {
+		

```

Comment:
What si the purpose of this Category? From what i see it is not an enumeration, since an enumeration relates  A Literal to a unique value. Here it looks to me, like the same could be achieved as an Array of StringProperty In RequirementsAttribute. 

I recommend to keep the model simple first and add new Categories when realy needed. Usually a migration to new categories is easier, than migrating from removed categories or replaced categories.

Reply:
I'm customizing this cateogry in the next pull request